ARVideoKit/Rendering/RenderAR.swift

```diff
@@ -8,33 +8,37 @@
 
 import Foundation
 import ARKit
-fileprivate var view:Any?
-fileprivate var renderEngine:SCNRenderer!
+
 @available(iOS 11.0, *)
-internal struct RenderAR {
-    internal var ARcontentMode:ARFrameMode!
-    init(_ ARview:Any?, renderer:SCNRenderer, contentMode:ARFrameMode) {
+struct RenderAR {
+    private var view: Any?
+    private var renderEngine: SCNRenderer!
```

Comment:
this will cause memory leaks. you may keep the private tag but make this variable global (outside the struct)